### PR TITLE
Cosmetic changes: removed .py suffix, fixed link in README

### PR DESCRIPTION
**Pull request by goldshtn**

### CODE REVIEW

**Comment by mcaleavya** [comment]
The removal of the .py makes this non standard with the rest of the current tools.

**Comment by goldshtn** [comment]
How so? Quoting from https://github.com/iovisor/bcc/blob/master/tools/biolatency.py:

```
# biolatency    Summarize block device I/O latency as a histogram.
#       For Linux, uses BCC, eBPF.
#
# USAGE: biolatency [-h] [-T] [-Q] [-m] [-D] [interval] [count]
```

and:

```
examples = """examples:
    ./biolatency            # summarize block I/O latency as a histogram
    ./biolatency 1 10       # print 1 second summaries, 10 times
    ./biolatency -mT 1      # 1s summaries, milliseconds, and timestamps
    ./biolatency -Q         # include OS queued time in I/O time
    ./biolatency -D         # show each disk device separately
"""
```

And quoting from https://github.com/iovisor/bcc/blob/master/man/man8/biotop.8:

```
.SH EXAMPLES
.TP
Summarize block device I/O by process, 1 second screen refresh:
#
.B biotop
.TP
Don't clear the screen:
#
.B biotop -C
.TP
5 second summaries, 10 times only:
#
.B biotop 5 10
```

File: README.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Examples:
 
 Tools:
 
-- tools/[argdist](tools/argdist.py): Display function parameter values as a histogram or frequency count. [Examples](tools/argdist_example.txt).
+- tools/[argdist](tools/argdist.py): Display function parameter values as a histogram or frequency count. [Examples](tools/argdist_examples.txt).
```
Comment (4ast) [inline README.md]: should be the other way around. all other tools do _example.txt

Comment (goldshtn) [inline README.md]: @4ast They do, but OTOH if you look at all the man files it says on the bottom to look at the corresponding _examples.txt file (plural, not singular). I don't mind either way... But it's not consistent already.

**Comment by goldshtn** [comment]
Renamed to *_examples.txt

File: README.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ Examples:
 
 Tools:
 
-- tools/[argdist](tools/argdist.py): Display function parameter values as a histogram or frequency count. [Examples](tools/argdist_example.txt).
+- tools/[argdist](tools/argdist.py): Display function parameter values as a histogram or frequency count. [Examples](tools/argdist_exampls.txt).
```
Comment (4ast) [inline README.md]: typo

Comment (goldshtn) [inline README.md]: Wow, I don't know what's wrong with me today. Fixed. Thanks.

**Pull request merged**